lib/app_profiler/request_parameters.rb

```diff
@@ -50,6 +51,12 @@ def valid?
       true
     end
 
+    def valid_backend?
+      return true if AppProfiler::Backend::StackprofBackend.name == backend
```

Comment:
I know it achieves the same outcome and it is probably a matter of personal preference - but I would rather get rid of `AppProfiler::Backend::StackprofBackend.name == backend` and have something which checks if `backend` is one of the two.  We can define a constant `AVAILABLE_BACKENDS`